src/main/java/lc/kra/system/keyboard/GlobalKeyboardHook.java

```diff
@@ -54,6 +56,10 @@
 	private boolean menuPressed, shiftPressed, controlPressed, winPressed, extendedKey;
 	
 	private List<GlobalKeyListener> listeners = new CopyOnWriteArrayList<GlobalKeyListener>();
+	
+	private Set<Integer> holdDownKeyCodes = new HashSet<Integer>();
```

Comment:
Using the past tense fits better to the library I think. So heldDownKeyCodes.

Reply:
done